package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "capital-framework",
-  "version": "4.7.0",
+  "version": "4.7.2",
```

Comment:
Ah, probably because you [bumped it in the feature PR](https://github.com/cfpb/capital-framework/pull/505/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R3) :)